flink-core/src/main/java/org/apache/flink/api/common/accumulators/Histogram.java

```diff
@@ -53,12 +53,7 @@ public TreeMap<Integer, Integer> getLocalValue() {
     public void merge(Accumulator<Integer, TreeMap<Integer, Integer>> other) {
         // Merge the values into this map
         for (Map.Entry<Integer, Integer> entryFromOther : other.getLocalValue().entrySet()) {
-            Integer ownValue = this.treeMap.get(entryFromOther.getKey());
-            if (ownValue == null) {
-                this.treeMap.put(entryFromOther.getKey(), entryFromOther.getValue());
-            } else {
-                this.treeMap.put(entryFromOther.getKey(), entryFromOther.getValue() + ownValue);
-            }
+            this.treeMap.merge(entryFromOther.getKey(), entryFromOther.getValue(), Integer::sum);
```

Comment:
can you confirm there is an existing  unit test for this code please.

Reply:
Looking at the javadoc for merge if entryFromOther.getValue() is null, then the entry is removed, is this possible?   
Testing the cases where null is our own Value (because the record does not exist) and null for the other value would be useful I think, as well as the happy path.